Bug: https://bugzilla.gnome.org/show_bug.cgi?id=602666
Author: Josselin Mouette <joss@debian.org>

The server binary refuses to start when it is not run on the same hardware as
the one where it was built on. This check is utterly and absolutely useless,
and makes it impossible to produce working binary packages.

Index: libgtop-2.28.0/src/daemon/server.c
===================================================================
--- libgtop-2.28.0.orig/src/daemon/server.c	2009-11-22 21:14:47.613184919 +0100
+++ libgtop-2.28.0/src/daemon/server.c	2009-11-22 21:15:02.673188074 +0100
@@ -86,16 +86,6 @@ main(int argc, char *argv[])
 			 LIBGTOP_COMPILE_RELEASE);
 		_exit (1);
 	}
-#else
-	if (strcmp (uts.sysname, LIBGTOP_COMPILE_SYSTEM) ||
-	    strcmp (uts.release, LIBGTOP_COMPILE_RELEASE) ||
-	    strcmp (uts.machine, LIBGTOP_COMPILE_MACHINE)) {
-		fprintf (stderr, "Can only run on %s %s %s\n",
-			 LIBGTOP_COMPILE_SYSTEM,
-			 LIBGTOP_COMPILE_RELEASE,
-			 LIBGTOP_COMPILE_MACHINE);
-		_exit (1);
-	}
 #endif
 
 	glibtop_init_p (glibtop_global_server, 0, 0);
